django/contrib/admin/options.py

```diff
@@ -962,12 +962,21 @@ def log_deletion(self, request, obj, object_repr):
             action_flag=DELETION,
         )
 
-    @display(description=mark_safe('<input type="checkbox" id="action-toggle">'))
+    @display(
+        description=mark_safe(
+            '<input type="checkbox" id="action-toggle" aria-label="Select all">'
+        )
+    )
     def action_checkbox(self, obj):
         """
         A list_display column containing a checkbox widget.
         """
-        return helpers.checkbox.render(helpers.ACTION_CHECKBOX_NAME, str(obj.pk))
+        attrs = {
+            "class": "action-select",
+            "aria-label": "Select row instance - %s" % obj,
```

Comment:
I think we should make clear that it's being selected for an action to be applied. Maybe...
```suggestion
            "aria-label": f"Select this object for an action - {obj}",
```

Or it might be better more natural, like: "Select the object '{obj}' for an action."

I am not sure 🤔

Reply:
That way its much better!

But a doubt, can we use the f strings feature in Django code? I think if we use fstrings we lose compatibilities with python versions older than 3.7, is that right?

Reply:
Django 5.0 supports only Python 3.10+.